public/maillog.php

```diff
@@ -0,0 +1,121 @@
+<?php
+/*
+* to do :   - add pagination
+*	          - sort by mounth
+* 	        - remove old logs
+*/
+
+require_once('common.php');
+
+authentication_require_role('admin');
+
+if (!is_dir(__DIR__ . '/../maillog')) {
+    die("../maillog does not exist");
+}
```

Comment:
The script should also die() if `$CONF['maillogging']` is not enabled.

BTW: config.inc.php should also set a default value for it (I'd tend to NO).